components/style/properties/helpers/animated_properties.mako.rs

```diff
@@ -390,12 +390,20 @@ impl Interpolate for BackgroundSize {
 impl Interpolate for RGBA {
     #[inline]
     fn interpolate(&self, other: &RGBA, progress: f64) -> Result<Self, ()> {
-        Ok(RGBA {
-            red: try!(self.red.interpolate(&other.red, progress)),
-            green: try!(self.green.interpolate(&other.green, progress)),
-            blue: try!(self.blue.interpolate(&other.blue, progress)),
-            alpha: try!(self.alpha.interpolate(&other.alpha, progress)),
-        })
+        let alpha = try!(self.alpha.interpolate(&other.alpha, progress));
+        alpha.min(1.);
```

Comment:
Thank you!  I am really unfamiliar with Rust.  Your review is really really helpful.

By the way, I also realized that we need to clamp the interpolated value because some cubic-bezier functions produce out of range [0, 1], e.g. http://cubic-bezier.com/#0,1,1,1.5

I will update this PR soon.